recon/src/recon/btreestore.rs

```diff
@@ -55,24 +54,16 @@ where
 
     /// Return the hash of all keys in the range between left_fencepost and right_fencepost.
     /// Both range bounds are exclusive.
-    pub fn hash_range(
-        &self,
-        left_fencepost: &K,
-        right_fencepost: &K,
-    ) -> anyhow::Result<HashCount<H>> {
-        if left_fencepost >= right_fencepost {
+    pub fn hash_range(&self, range: Range<&K>) -> anyhow::Result<HashCount<H>> {
+        if range.start >= range.end {
```

Comment:
My interpretation is that exclusivity wins, so if start and end are the same then the range is always empty, even a key equal to start is not included because end excludes it.